tests/find_cmd_tests.rs

```diff
@@ -12,8 +12,10 @@ use assert_cmd::Command;
 use predicates::prelude::*;
 use regex::Regex;
 use serial_test::serial;
+use std::collections::HashMap;
 use std::fs::{self, File};
 use std::io::{Read, Write};
+use std::os::unix;
```

Comment:
This has caused a test failure on the Windows platform. Could you take a look?
Operating system-specific `use` should be wrapped in `#[cfg(target_os)]`.